src/frontend/hooks/useSendInnSkjema.tsx

```diff
@@ -17,25 +18,34 @@ export const useSendInnSkjema = (): {
     const { soknadApiProxyUrl } = Miljø();
     const { valgtLocale } = useSpråk();
     const sendInnSkjemaV8 = async (): Promise<[boolean, ISøknadKontraktV8]> => {
-        settInnsendingStatus({ status: RessursStatus.HENTER });
-
-        const formatert: ISøknadKontraktV8 = dataISøknadKontraktFormatV8(valgtLocale, søknad);
-
-        const res = await sendInn<ISøknadKontraktV8>(
-            formatert,
-            axiosRequest,
-            `${soknadApiProxyUrl}/soknad/v8`,
-            (res: AxiosError) => {
-                const responseData = res.response?.data;
-                if (responseData && erModellMismatchResponsRessurs(responseData)) {
-                    settSisteModellVersjon(responseData.data.modellVersjon);
+        try {
```

Comment:
Trenger vi egentlig denne try-catch blokken? Mulig det er noe jeg ikke ser 😅

Reply:
Bra at du spør, det er jo ikke rett frem å se det ut i fra koden. 😃 Da vi hadde problemer med KS-søknaden var det fordi konvertering til riktig format (`const formatert: ISøknadKontraktV8 = dataISøknadKontraktFormatV8(valgtLocale, søknad);`) manglet en tekst fra Sanity. Det hele feilet, og vi fikk denne feilen i Sentry: [undefined is not an object (evaluating 'r.api_navn')](https://sentry.gc.nav.no/organizations/nav/issues/585058/?environment=www.nav.no&project=137&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d). Det virket ikke som en ille feil, og dermed ble den nedprioritert når andre ting virket viktigere.

Jeg legger her hele logikken rundt innsending i en try-catch for å plukke opp alle slike feil og sende de til Sentry med en bedre beskrivelse av alvorlighetsgraden. I tillegg logger vi hvis vi får nettverksproblemer. Ved nettverksfeil skal vi ikke throwe en ordentlig feil siden vi skal vise en custom feilmelding til brukeren (den skal ikke gå til ErrorBoundary). Derfor håndteres det separat med egen logging. 😊

Reply:
Sounds good! 🙌